build/common/utils.sh

```diff
@@ -66,3 +68,83 @@ function envsubst_create_config {
         envsubst < ${1} > /etc/nginx/conf.d/${application}.${2}.conf
     done
 }
+
+function postfix_config {
```

Comment:
is it possible to disable this container in case we use an external SMTP? Ensure it's explained in the README how to do this.

Reply:
Added a note in the README. :-)

Reply:
Added a section in README.